projects/ADuCM3029_demo_esp8266/app_src/ADuCM3029_demo_esp8266.c

```diff
@@ -0,0 +1,372 @@
+ *   @brief  Example of using esp8266 module to send adxl362 data
+ *   @author Mihail Chindirs (mihail.chindris@analog.com)
+********************************************************************************
+ * Copyright 2019(c) Analog Devices, Inc.
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *  - Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  - Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#include <sys/platform.h>
+#include "adi_initialize.h"
+#include <drivers/pwr/adi_pwr.h>
+#include <stdio.h>
+
+#include "parameters.h"
+
+#include "tcp_socket.h"
+#include "mqtt_client.h"
+#include "adxl362.h"
+#include "error.h"
+#include "wifi.h"
+#include "gpio.h"
+#include "irq.h"
+#include "irq_extra.h"
+#include "spi_extra.h"
+#include "delay.h"
+
+#define PRINT_ERR_AND_RET(msg, ret) do {\
+	printf("%s - Code: %d (-0x%x) \n", msg, ret, ret);\
+	return ret;\
+} while (0)
+
+int platform_init()
+{
+	/**
+	 * Initialize managed drivers and/or services that have been added to
+	 * the project.
+	 * @return zero on success
+	 */
+	adi_initComponents();
+
+	if (ADI_PWR_SUCCESS != adi_pwr_Init())
+		return 0;
```

Comment:
Usually return non-zero on error and 0 on success. But might want to use the power module for this.

Reply:
done